src/site/content/en/fast/font-best-practices/index.md

```diff
@@ -135,41 +84,34 @@ stylesheet to download.
 
     body {
         font-family: "Open Sans";
-
     }
+
+    ...etc.
+
   </style>
 </head>
-
 ```
 
 {% Aside %}
-    Inlining the font files themselves is not recommended. Inlining large
-    resources like fonts is likely to delay the delivery of the main document,
-    and with it, the discovery of other resources.
-
+    Note that if only some of the CSS is inlined, then the browser will still need to wait for all the CSS to be loaded, before it can discover if fonts are needed.<br>
+    <br>
+    Also note that inlining the font files themselves is not recommended. Inlining large resources like fonts is likely to delay the delivery of the main document, and with it, the discovery of other resources.
```

Comment:
Ope, nevermind. I see you mentioned it here.

Reply:
If the document is streaming, and it's a large chunk of BASE64, then it delays the rest of the doc arriving, which means the preload scanner can't get stuck on it.

Plus it says "is likely to delay". Likely being the key here :-)